pro/src/screens/OfferIndividual/StocksEvent/StocksEvent.tsx

```diff
@@ -262,8 +263,8 @@ const StocksEvent = ({ offer }: IStocksEventProps): JSX.Element => {
   })
 
   const isFormEmpty = () => {
-    return formik.values.stocks.every(
-      val => val === STOCK_EVENT_FORM_DEFAULT_VALUES
+    return formik.values.stocks.every(val =>
+      isEqual(val, STOCK_EVENT_FORM_DEFAULT_VALUES)
```

Comment:
(Suggestion) On ne pourrait pas utiliser `Object.is` ici ? Au quel cas on pourrait se passer d'ajouter lodash

Reply:
Je ne connaissais pas `Object.is` ! mais malheureusement ça ne marche pas, ça compare les références :/

Reply:
Yep en effet après avoir relu de la doc ça ne me semble pas correct ici (ma première source était pas bonne 😬)